lib/api.js

```diff
@@ -577,6 +577,22 @@ api.declare({
 });
 
 /**
+ * List AMIs and their usage
+ */
+api.declare({
+  method: 'get',
+  route: '/internal/ami-usage',
+  name: 'amiUsage',
+  title: 'See the list of AMIs and their usage',
+  stability: API.stability.experimental,
+  scopes: [['ec2-manager:internals']],
+  description: 'Lists out all of the AMIs and their usage',
+}, async function(req, res) {
+  let amiUsage = await this.state.listAmiUsage();
```

Comment:
It has been a standard in this project, so it is okay, but bear in mind that in general, variables should be `const` by default and we only use `let` for cases where we have good reasons to modify them (like the control variable in a `for` loop).

Reply:
Good to know, thanks for the info!

Reply:
Good point.